src/core/service/notifications/jobs/handler.go

```diff
@@ -87,3 +87,13 @@ func (h *Handler) HandleReplication() {
 		return
 	}
 }
+
+// HandleReplicationTask handles the webhook of replication task
+func (h *Handler) HandleReplicationTask() {
+	log.Debugf("received replication task status update event: task-%d, status-%s", h.id, h.status)
+	if _, err := dao.UpdateTaskStatus(h.id, h.status); err != nil {
```

Comment:
Use the Execution Manager defined here: https://github.com/goharbor/harbor/blob/replication_ng/src/replication/ng/replication.go#L34

Reply:
i will change.